src/App.tsx

```
@@ -16,46 +17,106 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  // eslint-disable-next-line
+  let [ value, setValue ] = useState('');
+  // eslint-disable-next-line
+  let [reverse, setReverse] = useState(false);
```

do not ignore errors, try to fix them